src/cmds/qsub.c

```diff
@@ -4919,12 +4898,32 @@ do_submit(char *retmsg)
 	char *new_jobname = NULL;
 	int rc;
 	char *errmsg;
+	int retries;
 
-    rc = parse_dfltqsubargs(retmsg);
-	if(rc != 0)
-	    return (rc);
+    if (dfltqsubargs != NULL) {
+		/*
+		 * Setting options from the server defaults will not overwrite
+		 * options set from the job script. CMDLINE-2 means
+		 * "one less than job script priority"
+		 */
+		for (retries = 2; retries > 0; retries--) {
+			rc = do_dir(dfltqsubargs, CMDLINE - 2, retmsg, MAXPATHLEN);
+			if (rc >= 0)
+				break;
+			if (retries == 2)
+				refresh_dfltqsubargs();
+		}
+		if (rc != 0)
+			return (rc);
+	}
 
+	/* 
+	 * get environment variable if -V option is set. Return the code
+	 * VOPT_SET if -V option is detected in background qsub.
+	 */
 	if(V_opt) {
+		if(is_background)
+			return VOPT_SET;
```

Comment:
Also, it would be better if we can check this in do_daemon_stuff and exit the daemon in this return case. This way future qsubs will not have to continuously consult with the background qsub just to know that it should not be going to the background